pom.xml

```diff
@@ -160,6 +167,11 @@
                 <artifactId>validation-api</artifactId>
                 <version>${version.javax.validation}</version>
             </dependency>
+            <dependency>
+                <groupId>jakarta.validation</groupId>
+                <artifactId>jakarta.validation-api</artifactId>
```

Comment:
Let's also swap these dependency declarations to stick to the consistent (alphabetic) sorting.